spec/lib/terraforming/resource/efs_spec.rb

```diff
@@ -0,0 +1,110 @@
+require "spec_helper"
+require 'awesome_print'
+
+module Terraforming
+  module Resource
+    describe EFS do
+      let(:client) do
+        Aws::EFS::Client.new(stub_responses: true)
+      end
+
+      let(:efs_description_0) do
+          {
```

Comment:
__[rubocop]__
* Use 2 (not 4) spaces for indentation. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)

<!-- Comment by [SideCI](https://sideci.com) -->